source/reference/array.rst

```diff
@@ -236,7 +236,200 @@ Unevaluated Items
 
 |draft2019-09|
 
-Documentation Coming Soon
+The ``unevaluatedItems`` keyword selects any data types not evaluated
```

Comment:
(Keeping in mind for this and any other comments that I'm honestly not personally too familiar with conventions used throughout the book here, so if you see something I say contradicted somewhere feel free to say so!)

"values" is more correct than "data types" -- and instead of using the verb "selects" I'd probably use "applies validation to" or simply "applies to".